cfgov/jinja2/v1/_includes/organisms/search-bar.html

```diff
@@ -49,7 +49,7 @@ <h4 class="u-mb0">
         <div class="o-search-bar_input o-search-bar_col">
             <div class="input-with-btn input-with-btn__bar">
                 <div class="input-with-btn_input {% if value.autocomplete %} m-autocomplete{% endif %}">
-                    <input type="text" name="q" id="q" >
+                    <input class="a-text-input" type="text" name="q" id="q" >
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/pull/3120/commits/78cf4b91c3a9c0e454943d540d44cad8c1e190b6